### ISSUE TITLE
[Question] Support for Splunk ITSI

### ISSUE THREAD

**Issue by rockaut** [ansible-collections/splunk.es#24]
Heja there,

sorry to bother you with this questions: I am currently building some modules around the ITSI API along-side our installation here. Now I understand, that those would not fit into this collection so something like splunk.itsi would be better. I also understand that community driven collections are prefixed with community.* and vendor driven with vendor.* but obviously we would love to see an vendor collection we can contribute to.
Long story short: do you have any @ for me where how we could start that collection?

Or should I just poke @gundalow for creating an community one? *wave*

**Comment by rockaut** [comment]
closing this as we most likely to create a community.splunk repo instead.
